### ISSUE TITLE
hi, the drop and drag feature does not work on IE

### ISSUE THREAD

**Issue by ziyiwang** [JeremyFagis/dropify#87]
hi, the drop and drag feature does not work on IE

IE defaults to open a file inside browser

**Comment by Majusek** [comment]
Hi,

do you resolve the problem?

Best regards

**Comment by AnselmoAlexandre** [comment]
Not yet mate.
Still want it to work but, its not working still.
Have you some ideias?

**Comment by Majusek** [comment]
No, but I seen another problem, button "remove" doesn't work too.

Do you have the same problem with this button?

**Comment by AnselmoAlexandre** [comment]
No, for me the remover button works just fine. Check the docs

**Comment by The4thLaw** [comment]
This seems to be because Dropify is "just" an overlay over a regular input element, and IE doesn't support drag-and-dropping on file inputs to set their value. With the current architecture, it will never work on IE.

The fact that the library doesn't work on IE could be a design choice (it's up to the author), but I think it should be documented in the README.
